acp/TERMINOLOGY.md

```diff
@@ -0,0 +1,9 @@
+## ACP Terminologies
+- 'SourceHub Address' is a Cosmos Address with a specific SourceHub prefix.
+- 'Identity' is a combination of SourceHub Address and a Key-Pair Signature.
+- 'DPI' means 'DefraDB Policy Interface'.
+- 'Partially-DPI' policy means a policy with atleast one DPI compliant resource.
+- 'Permissioned Schema' means to have a policy on the schema, like: `@policy(id:".." resource: "..")`
```

Comment:
todo: Schema do not have policies in this PR - please change all references like this to `Permissioned Collection`

Reply:
> todo: Schema do not have policies in this PR - please change all references like this to `Permissioned Collection`.

This language is somewhat confusing for me still. When we link the Policy it is linked in a `Add Schema` command. I understand schema refers to global and collection more local.

question: When a migration happens, would that remove the policy on that new `schema/collection`?  

- [ ] Update to `Permissioned Collection` 


> todo: Please also move the cli/http commands so that they are not on `schema`, as they do not affect schema, but collections.

Sorry can you clarify which commands? and where to move to? Are you referring to file names?

Reply:
> When we link the Policy it is linked in a Add Schema command.

The naming around adding stuff via an SDL is still confusing, it might be that `add schema` needs to be renamed or the documentation expanded.

> question: When a migration happens, would that remove the policy on that new schema/collection?

No.

> Sorry can you clarify which commands? and where to move to? Are you referring to file names?

I think you used a command in your demo that doesn't exist here - I think you had a `schema add policy` like command - is why added this comment - reading the CLI changes here however it looks like it doesn't exist :grin: So don't worry about this part of the comment lol (unless other branches you have add such a command)

Reply:
> The naming around adding stuff via an SDL is still confusing, it might be that `add schema` needs to be renamed or the documentation expanded.

Agreed, I think the explanation of `shape` vs `schema` vs `collection` and `global` or `local` terminologies is very important in the documentation.

> > Sorry can you clarify which commands? and where to move to? Are you referring to file names?
> 
> I think you used a command in your demo that doesn't exist here - I think you had a `schema add policy` like command - is why added this comment - reading the CLI changes here however it looks like it doesn't exist 😁 So don't worry about this part of the comment lol (unless other branches you have add such a command)

I think you might be mistaken, there is no extra command that was used in the demo, you can see all the commands here: https://source.almanac.io/docs/acp-v0-implementation-demo-b4fff715fbe24344909952f2b763fd02

The command you might be thinking was the normal `defradb client schema add` command but the schema was linking to a policy using the `@policy` directive, here is the exact command I used:

```graphql
defradb client schema add '
type Users @policy(
    id: "24ab8cba6d6f0bcfe4d2712c7d95c09dd1b8076ea5a8896476413fd6c891c18c",
    resource: "users"
) {
    name: String
    age: Int
}
'
```